contentcuration/contentcuration/static/js/edit_channel/details/views.js

```diff
@@ -0,0 +1,475 @@
+var Backbone = require("backbone");
```

Comment:
Channel details panel- implemented so that adding this to topics should be straightforward